src/shared/components/palette/base-field/field-write.component.ts

```diff
@@ -35,7 +35,8 @@ export class FieldWriteComponent extends AbstractFieldWriteComponent implements
   }
 
   ngOnInit(): void {
-    let componentClass = this.paletteService.getFieldComponentClass(this.caseField, true);
+    let writeMode = !(this.caseField.display_context === 'READONLY')
+    let componentClass = this.paletteService.getFieldComponentClass(this.caseField, writeMode);
```

Comment:
@maliumer - please can you add a basic unit test to `field-write.component.spec.ts` to check that `paletteService.getFieldComponentClass()` is being called with the correct parameters, based on the value of `writeMode`? In fact, you could probably do away with the separate `writeMode` variable - since it's a boolean - and just use the expression as the second argument to `getFieldComponentClass()`